mixins/selectable/test/selectable-mixin.test.js

```diff
@@ -0,0 +1,66 @@
+import { defineCE, expect, fixture, oneEvent } from '@open-wc/testing';
+import { html, LitElement } from 'lit-element/lit-element.js';
+import { SelectableMixin } from '../selectable-mixin.js';
+
+const tag = defineCE(
+	class extends SelectableMixin(LitElement) {
+		render() {
+			return html`
+				${this._renderCheckbox()}
+				${this._renderSelectAction()}
+			`;
+		}
+	}
+);
+
+describe('SelectableMixin', () => {
+	it('Sets checked status to undefined when no key is given', async() => {
+		const element = await fixture(`<${tag} selected="true"></${tag}>`);
+		expect(element.selected).to.be.undefined;
+	});
+
+	describe('Events', () => {
+		let element;
+		beforeEach(async() => {
+			element = await fixture(`<${tag} key="1234" selectable></${tag}>`);
+		});
+		it('dispatches custom event when checkbox is checked', async() => {
+			// simulate a checkbox click
+			setTimeout(() => {
+				const checkbox = element.shadowRoot.querySelector('.d2l-input-checkbox');
+				checkbox.checked = true;
+				checkbox.dispatchEvent(new Event('change'));
+			});
+			const { detail } = await oneEvent(element, element._selectedEventName);
+			expect(detail.key).to.equal('1234');
+			expect(detail.selected).to.equal(true);
```

Comment:
also need test to show state could change from selected to unselected

Reply:
Tests are missing assertion on initial `element.selected` state and after the "click"